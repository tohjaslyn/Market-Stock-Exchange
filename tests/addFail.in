add trader1 100
add trader1 10
add trader2 -1
exit
