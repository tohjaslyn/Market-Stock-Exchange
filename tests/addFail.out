$ Success.
$ Trader with given ID already exists.
$ Initial balance cannot be negative.
$ Have a nice day.
